src/neo4j_graphrag/llm/mistralai_llm.py

```diff
@@ -84,10 +84,11 @@ def invoke(self, input: str) -> LLMResponse:
                 messages=self.get_messages(input),
                 **self.model_params,
             )
-            if response is None or response.choices is None or not response.choices:
-                content = ""
-            else:
-                content = response.choices[0].message.content or ""
+            content: str = ""
```

Comment:
Regarding this, I think we should wait a bit, because they've changed this part of the code again last week. So maybe just pin the version to before the pb first arose, and come back to it when their code is stabilized. WDYT?

Reply:
Agreed